### ISSUE TITLE
Actions Table

### ISSUE THREAD

**Issue by MattH2023** [SafetyCulture/safetyculture-exporter#405]
We extract your data into SQL tables, we have recently noticed that you have added ‘action_label’ and ‘deleted’ into your Actions table. 
This of course would be extremely helpful to us, however there doesn’t appear to be any data returning for these new columns when we download the data. 
Are they just place holders for now or should we be seeing data?

**Comment by BCook98** [comment]
Hi @MattH2023 thanks for reaching out, both `action_label` and `deleted` should return valid data when exporting. 

Can you share what command you are running to export data `safetyculture-exporter csv` or `safetyculture-exporter sql`.
Could you also confirm what version of the exporter you are running, you can get this by running `./safetyculture-exporter --version` on linux or `.\safetyculture-exporter.exe --version` on Windows.

**Comment by MattH2023** [comment]
Hiya

We run ./iauditor-exporter sql
Version: iauditor-exporter version 3.9.3

Since its now called ‘SafetyCulture Exporter’ I’m assuming we are using the old version? 
would this cause the issue

**Comment by BCook98** [comment]
That's correct, you'll have to upgrade to the latest version in order to get access to these fields. 

Although v4 was a major upgrade the only breaking change is the rename from `iauditor-exporter` to `safetyculture-exporter`. You'll also need to rename the config file `iauditor-exporter.yaml` to `safetyculture-exporter.yaml`.

**Comment by MattH2023** [comment]
Thank you, i will take this back to our Admin team.

**Comment by BCook98** [comment]
Cheers @MattH2023 I'll close this out for now.
